### PR TITLE
Dart Debug Extension: don't forward events to Chrome that will cause a crash

### PR DESCRIPTION
**Pull request by elliette**
See details in https://bugs.chromium.org/p/chromium/issues/detail?id=1469092

This PR skips forwarding events that will cause a crash to Chrome, and instead displays a warning to the user:

Screenshot 2023-08-02 at 3 18 05 PM

FYI @DanTup

### CODE REVIEW

File: dwds/debug_extension_mv3/web/debug_session.dart [hunk 1]
```diff
@@ -466,6 +479,55 @@ void _forwardDwdsEventToChromeDebugger(
...
 }
 
+bool _shouldSkipEventForChrome115Bug(String command) {
+  final unsupportedOnChrome115 = command.contains('Debugger.setBreakpoint') ||
+      command.contains('Debugger.pause');
+  if (unsupportedOnChrome115) {
+    final chromeVersionMatch =
+        RegExp('Chrome/([0-9.]+)').firstMatch(window.navigator.userAgent);
+    final chromeVersion =
+        chromeVersionMatch == null ? '' : chromeVersionMatch[0];
```
Comment (annagrin) [inline dwds/debug_extension_mv3/web/debug_session.dart:489]: Nit: can this be `chromeVersion = chromeVersionMatch?[0]` ?

**Review by annagrin — APPROVED** [review]
Approved with minor comments

File: dwds/debug_extension_mv3/web/debug_session.dart (hunk 1 above)
Comment (elliette) [reply dwds/debug_extension_mv3/web/debug_session.dart]: Good point, done!

File: dwds/debug_extension_mv3/web/debug_session.dart [hunk 2]
```diff
@@ -466,6 +479,55 @@ void _forwardDwdsEventToChromeDebugger(
...
+    if (chromeVersion?.startsWith('Chrome/115') ?? false) {
```
[the remark this answers is not in the document]
Comment (elliette) [reply dwds/debug_extension_mv3/web/debug_session.dart]: Done!

**Pull request merged**